_posts/2021-05-21-js-waku.md

```diff
@@ -0,0 +1,171 @@
+2021-05-21-js-waku.md---
+layout: post
+name:  "Presenting JS-Waku: Waku v2 in the browser"
+title:  "Presenting JS-Waku: Waku v2 in the browser"
+date:   2021-05-21 12:00:00 +0800
+author: franck
+published: true
+permalink: /2021-05-21-js-waku
+categories: platform
+summary: "We are summarizing the Waku v2 status, introducing JS-Waku, what we achieved so far and what is next."
```

Comment:
This will show up as a twitter card preview, not very enticing and descriptive which may limit reach. Focus on running in the browser etc. This post is also not really summarizing Waku v2 status in more global terms, that's another post.